### DISCUSSION TITLE
chrome cmd arguments

### DISCUSSION THREAD

**Discussion by ghost** [yandeu/five-server#27] [General]
I'm not sure the five-server understands command line arugments in the `browser` section of configuration file, but I would be really grateful if you could add things like this. 

```bash
# .fiveserverrc
....
"browser":"chrome --auto-open-devtools-for-tabs"
...
```

Like I need to open chrome with that argument for automatic dev tools.

**Comment by ghost** [comment]
I figured that out, actually that works fine with args .

**Reply by yandeu** [reply]
👍🏻
